### PR TITLE
475 reactivate linux build

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 
+version: '{build}'
+
 # Download script file from GitHub
 init:
-  ps: |
-    $ErrorActionPreference = "Stop"
-    Invoke-WebRequest http://raw.github.com/krlmlr/r-appveyor/master/scripts/appveyor-tool.ps1 -OutFile "..\appveyor-tool.ps1"
-    Import-Module '..\appveyor-tool.ps1'
+  - ps: Update-AppveyorBuild -Version "$($env:ospsuite_version).$($env:appveyor_build_version)"
```
Comment (msevestre) [inline appveyor.yml:7]: @Yuri05 Unrelated but ensure that we are using the global version (suite component)

File: rakefile.rb [hunk 2]
```diff
@@ -11,12 +11,11 @@
 
 task :prepare_for_build, [:product_version] do |t, args|
   product_version = sanitized_version(args.product_version)
- 
   copy_files_to_lib_folder
 
   update_package_version(product_version)
 
-  install_pksim('feature/relative_expression_redesign')
+  install_pksim('develop')
```
Comment (msevestre) [inline rakefile.rb:18]: @Yuri05 Done

File: rakefile.rb [hunk 3]
```diff
@@ -168,9 +168,9 @@ def sanitized_version(version)
 def update_package_version(version) 
   #Replace token Version: x.y.z with the version from appveyor
   replacement = {
-    /Version: \d\.\d\.\d/ => "Version: #{version}"
+    /Version: \d+\.\d+\.\d+/ => "Version: #{version}"
```
Comment (msevestre) [inline rakefile.rb:171]: Reged. .was parsing 9.x.z but not 10.x.z...

Comment (Yuri05) [reply rakefile.rb:171]: lol

**Pull request merged**